packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Banner/configAggregator.js

```diff
@@ -0,0 +1,76 @@
+import {
+    getMargin,
+    getBackgroundImages,
+    getBorder,
+    getPadding,
+    getTextAlign,
+    getCssClasses
+} from '../../utils';
+
+/**
+ * Determine the button type based on class
+ *
+ * @param node
+ * @returns {string}
+ */
+const getButtonType = node => {
+    if (node.classList.contains('pagebuilder-button-secondary')) {
+        return 'secondary';
+    }
+    if (node.classList.contains('pagebuilder-button-link')) {
+        return 'link';
+    }
+    return 'primary';
+};
+
+export default (node, props) => {
+    const wrapperElement = node.querySelector('[data-element="wrapper"]');
+    const overlayElement = node.querySelector('[data-element="overlay"]');
+    const linkElement = node.querySelector('a[data-element="link"]');
+    const buttonElement = node.querySelector('[data-element="button"]');
+    const showButton = node.getAttribute('data-show-button');
+    const showOverlay = node.getAttribute('data-show-overlay');
+
+    let minHeight = wrapperElement.style.minHeight
```

Comment:
Logical OR would improve readability.  `wrapperElement.style.minHeight || null;`

Reply:
Also has potentially unused additional logic and data processing; this could be wrapped in an else

Reply:
I've updated this logic to something which feels cleaner, thanks for reminding me logical OR exists!